### PR TITLE
nlohmann_json: add 3.7.3

### PR DESCRIPTION
**Pull request by theodelrieu**
Specify library name and version:  **nlohmann_json/3.7.1** (3.7.2 and 3.7.3 too)

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by Croydon** [comment]
Is there a specific need from your side for `3.7.1` and `3.7.2`? If not I would recommend to skip adding patch versions which are already old right now

**Comment by theodelrieu** [comment]
@Croydon I have no such need, I've added them for completeness.

Though we will have old patch versions all the time in the future.

**Comment by theodelrieu** [comment]
@danimtb What do you think about only keeping 3.7.3?

**Comment by danimtb** [comment]
yes, let's just keep the latest one if there is no need for the previous. We could add them later if those are requested. Thanks!

File: recipes/nlohmann_json/config.yml [hunk 1]
```diff
@@ -2,3 +2,9 @@
 versions:
   "3.7.0":
     folder: all
+  "3.7.1":
+    folder: all
+  "3.7.2":
+    folder: all
```
Comment (danimtb) [inline recipes/nlohmann_json/config.yml]: Remove this too!

**Pull request merged**